### PR TITLE
Oncoprint and co-occurrence numbers and stats

### LINKED ISSUE #408: Updated manuscript: Double check oncoprint Ns
#### What section of the manuscript should be updated?

Results section

#### Proposed changes

With the change in independent samples, we should re-check the oncoprint Ns and stats

### LINKED ISSUE #410: Updated manuscript: check co-occurrence Ns and stats
#### What section of the manuscript should be updated?

Results - we should check these numbers due to change in independent specimens

### PR DESCRIPTION
**Pull request by sjspielman**
Closes #408 
Closes #410 

This PR updates numbers/stats for oncoprint and co-occurrence results. The specific changes are detailed in the linked issues, but there are a few places where I want to discuss how to implement changes - 

1. https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/4cc47c788598f4cc73bfd5893513235ba0fa2084/content/03.results.md?plain=1#L144

For this, I see 2 _SMARCB1_ deletions [here](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/732d07cd2946849c6b5809e364043b390f85a2ed/analyses/oncoprint-landscape/tables/cancer_group_counts/primary_only/atypical-teratoid-rhabdoid-tumor_oncoprint_alteration_counts.tsv)
 But, I'm not sure [the figure panel](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/d8d0f04e017b9ec1b33b88051abdbe9423f95161/figures/pdfs/fig2/panels/primary_only_embryonal_oncoprint.pdf) shows this clearly? 
I'd appreciate another set of eyes there!

2. https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/789f65de824dcc28688d2751c0978bc6f86cb805/content/03.results.md?plain=1#L155
I wholesale removed _KRAS_ from this sentence as it is no longer in the figure, but the numbers still work out to 6/60.

3. https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/4cc47c788598f4cc73bfd5893513235ba0fa2084/content/03.results.md?plain=1#L171
I rephrased this sentence to open by saying _embryonal tumors_, since that's where we get the stats from: https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/732d07cd2946849c6b5809e364043b390f85a2ed/analyses/interaction-plots/results/cooccur_top50.Embryonal-tumor.tsv
But, I keep the other parts of the sentence saying MB since that's what the linked papers show. This seems fine as most embryonal samples we have are MB.

### CODE REVIEW

**Comment by jharenza** [comment]
>For this, I see 2 SMARCB1 deletions [here](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/732d07cd2946849c6b5809e364043b390f85a2ed/analyses/oncoprint-landscape/tables/cancer_group_counts/primary_only/atypical-teratoid-rhabdoid-tumor_oncoprint_alteration_counts.tsv)
But, I'm not sure [the figure panel](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/d8d0f04e017b9ec1b33b88051abdbe9423f95161/figures/pdfs/fig2/panels/primary_only_embryonal_oncoprint.pdf) shows this clearly?
I'd appreciate another set of eyes there!

Ah thanks, I think if we don't write about the deletions we detected, then this will be a flag, as these are the canonical two genes driving ATRT - we don't have the tables in supplement - wonder if we can somehow refer to the module? Not sure - it won't be in the figure bc it is only top N mutated.

>I wholesale removed KRAS from this sentence as it is no longer in the figure, but the numbers still work out to 6/60.

👍 


>I rephrased this sentence to open by saying embryonal tumors, since that's where we get the stats from: https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/732d07cd2946849c6b5809e364043b390f85a2ed/analyses/interaction-plots/results/cooccur_top50.Embryonal-tumor.tsv
But, I keep the other parts of the sentence saying MB since that's what the linked papers show. This seems fine as most embryonal samples we have are MB.

Thanks, I think I might have done this since the only emb tumors with the alteration are MB, which we can also say another way - say embryonal tumors ...blah blah.., this is driven by MB. Will review now!

File: content/03.results.md
Comment (jharenza) [inline content/03.results.md:137]: 👍

File: content/03.results.md
Comment (jharenza) [inline content/03.results.md:148]: 👍  for TP53, H3, but I think ATRX will need a denominator change. Still looks like 7+11 = 18.

File: content/03.results.md
Comment (jharenza) [inline content/03.results.md:156]: 👍

File: content/03.results.md
Comment (jharenza) [inline content/03.results.md:157]:
```suggestion
We observed rare fusions in _ERBB4_, _YAP1_, and/or _QKI_ in 10% (6/60) of ependymoma tumors.
```

I am going to update this to these genes since YAP1 fuses with MAML2 and MAMLD1, to not double count.

File: content/03.results.md
Comment (jharenza) [inline content/03.results.md:167]:
```suggestion
We analyzed mutational co-occurrence across the OpenPBTA, using a single tumor from each patient with available WGS (N = 668 patients). <!--SAMPLECOUNT-->
``` 
per issue #410

File: content/03.results.md
Comment (jharenza) [inline content/03.results.md:169]: 👍

File: content/03.results.md
Comment (jharenza) [inline content/03.results.md:171]:
```suggestion
In embryonal tumors, mutations in _CTNNB1_ significantly co-occurred with mutations in _TP53_ (OR = 43.6 95% CI: 7.1 - 265.8, q = 1.52e-3) as well as with mutations in _DDX3X_ (OR = 21.4, 95% CI: 4.7 - 97.9, q = 4.15e-3).
These events were driven by medulloblastomas and have been previously reported as significantly mutated in this tumor type [@doi:10.1038/nrc3410; 10.1200/JCO.2010.31.1670].
```
How about this?

File: content/03.results.md
Comment (jharenza) [inline content/03.results.md:173]: 👍

**Review by jharenza — CHANGES_REQUESTED** [review]
Wow, thanks for making this review so easy by outlining the results in the ticket! Made some comments for changes needed, and still up for discussion, SMARCB1.

File: content/03.results.md
Comment (sjspielman) [reply content/03.results.md:171]: I like it!

Comment (jharenza) [reply content/03.results.md]: possibly I messed up that DOI piece...maybe no space?

File: content/03.results.md
Comment (sjspielman) [reply content/03.results.md:148]: Good catch, thanks! I see 18/63 as well.

Comment (sjspielman) [reply content/03.results.md]: Fixed in https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/414/commits/52beeebb45b59275ce0994b1316ff76223aaa3e5.

**Comment by sjspielman** [comment]
> wonder if we can somehow refer to the module? Not sure - it won't be in the figure bc it is only top N mutated.

A quickly-achievable middle ground that I can definitely do before the end of today might be to add a sheet to Table S2 (DNA results) essentially comprised of all these files concatenated - https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/oncoprint-landscape/tables/cancer_group_counts/primary_only
Then we can say it's not depicted in the figure since it's not in the top N for the _overall cancer group_, and then just link the relevant sheet.

**Comment by jharenza** [comment]
>A quickly-achievable middle ground that I can definitely do before the end of today might be to add a sheet to Table S2 (DNA results) essentially comprised of all these files concatenated - https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/oncoprint-landscape/tables/cancer_group_counts/primary_only
Then we can say it's not depicted in the figure since it's not in the top N for the overall cancer group, and then just link the relevant sheet.

that would work!

**Comment by sjspielman** [comment]
Ok, the supp table is now ready for review next door in https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1689! 
I opened this issue #415 to add it to the MS, so let's circle back to the ATRT text for that issue once _for sure_ the updated Table S2 is into the analysis repo.

**Review by jharenza — APPROVED** [review]
Reviewed [#1689](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1689), so I guess we should also update the hash here in results/supplemental once more after that table is merged. Will approve this with that caveat and that Table S2 gets mentioned!

**Comment by sjspielman** [comment]
> Reviewed https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1689, so I guess we should also update the hash here in results/supplemental once more after that table is merged. Will approve this with that caveat and that Table S2 gets mentioned!

Yes, I envision that happening in #415, since the tables PR has to go through CI so won't be merged for a few hours, and in the meantime we can move this one along. Is that ok with you to just merge this as is?

**Comment by jharenza** [comment]
>Is that ok with you to just merge this as is? (are we saying the same thing?)

Ok with me!

File: content/03.results.md
Comment (sjspielman) [reply content/03.results.md]: Noting this is being fixed in the PR where I'm adding in Table S2 to the manuscript.

**Pull request merged**
